source/Patches/PlayerController.cs

```diff
@@ -35,9 +35,9 @@ public static void Update_Postfix(PlayerControllerB __instance)
                 //If we're looking backwards, offset the camera to the side ('leaning')
                 cameraOffset.x = Mathf.Sign(lookFlat.x) * ((70f - angleToBack)/70f);
             }
-        }
 
-        __instance.gameplayCamera.transform.localPosition = cameraOffset;
+            __instance.gameplayCamera.transform.localPosition = cameraOffset;
```

Comment:
I am making a big assumption that the camera position doesn't need to be updated outside of a vehicle here, but please let me know if my assumption is wrong!